### PR TITLE
Feature/textfield tweaks

### PR DESCRIPTION
**Pull request by mariannemagnussen**
https://www.figma.com/file/DY2nAi4QVg4DZWQSGP1A0n/Text-field?node-id=2%3A2

Updates:
- Border colors, colors and background-colors
- Paddings 
- Label for compact version
- Help text alignment = right by default

### CODE REVIEW

File: src/colors.pcss [hunk 1]
```diff
@@ -65,70 +65,64 @@ Our accent palette is used in the following applications: call to action buttons
...
+  --pink2: #ff01a8;
 
-    --deep-purple-rgb: 56, 3, 84;
-    /* Added for business table and checkbox - htf are we supposed to name these colors?*/
-    --medium-light-grey: #EBEBEB;
-    --medium-dark-grey: #595959;
-    --even-darker-grey: #222222;
+  --deep-purple-rgb: 56, 3, 84;
+  /* Added for business table and checkbox - htf are we supposed to name these colors?*/
+  --accent-250: #ebebeb;
```
Comment (HeleneGrini) [inline src/colors.pcss:125]: Må oppdatere der `--medium-light-grey` blir brukt andre steder i css-en

File: src/molecules/TextField/TextField.pcss [hunk 2]
```diff
@@ -120,35 +133,42 @@
...
+  .Icon {
+    fill: var(--grey-300);
+  }
 }
 
 .telia-textfield--compact {
   margin-top: 8px;
+  font-size: 14px;
 
+  //  Don't need the commented-out code after a design update on the compact label.
```
Comment (HeleneGrini) [inline src/molecules/TextField/TextField.pcss:159]: these comments are not allowed in css blocks. Is it working for you? 🤔   should use `/* */` instead

**Review by HeleneGrini — CHANGES_REQUESTED** [review]
The QuickSearch style is not working with these changes

File: src/molecules/TextField/TextField.pcss (hunk 2 above)
Comment (mariannemagnussen) [reply src/molecules/TextField/TextField.pcss]: nope 😅 Forgot to push the last commit 😬

**Review by HeleneGrini — CHANGES_REQUESTED** [review]
QuickSearch style is still broken

**Review by HeleneGrini — APPROVED** [review]
nais! 😎

**Pull request merged**